337 editorial ahead of FPWD

dr-shorthair opened the pull request:
Closes #337 , #332

Preview https://raw.githack.com/w3c/sdw-sosa-ssn/337-update-front-matter/ssn/index.html

dr-shorthair committed:
Rearrange System diagram to show link from Procedure to Property

dr-shorthair committed:
Merge branch 'gh-pages' into 332-tidy-up-diagrams

dr-shorthair committed:
add link System-->ExecutionCollection

dr-shorthair committed:
Begin revision of front matter for FPWD

dr-shorthair committed:
Fix Luis affiliation

dr-shorthair committed:
Missing buttons for class="ssn"
Consistent IDs for figures and links

dr-shorthair committed:
Missing comma in config.js

dr-shorthair committed:
'ing' forms for Procedures

dr-shorthair committed:
Parenthesize ref

dr-shorthair committed:
Merge branch 'gh-pages' into 337-update-front-matter

dr-shorthair committed:
Merge branch 'gh-pages' into 332-tidy-up-diagrams

dr-shorthair committed:
Merge branch '332-tidy-up-diagrams' into 337-update-front-matter

dr-shorthair committed:
Two broken Figure IDs

dr-shorthair committed:
Fix some broken links
Update System/Deployment figure
Other minor changes

dr-shorthair committed:
Add OGC doc number
Add note re untested terms
Change "2025 edition" to "2023 Edition" throughout
Fix some dud links

dr-shorthair committed:
More dud links fixed

dr-shorthair committed:
New OGC logo

dr-shorthair committed:
Remove stray </section> tags
Remove hide SSN axioms buttons

dr-shorthair committed:
Format config.js
Fix last broken link (!)

dr-shorthair committed:
Nesting sections in OMS extension

dr-shorthair committed:
OMS links

dr-shorthair committed:
Property no longer a superclass of Observable/ActuatableProperty

dr-shorthair committed:
Add ontology as well as paper for SSNX refs

dr-shorthair committed:
test dfn method for x-referencing definitions (from ReSpec)

dr-shorthair committed:
Replace anchors with <dfn> for all terms;
simplify links for definitions

dr-shorthair committed:
some <a> should be <dfn>

dr-shorthair committed:
Reinstate OGC logo which works

dr-shorthair committed:
Revert "some <a> should be <dfn>"

This reverts commit 8be7186d98ca35e780cf373347b570613428d9f3.

dr-shorthair committed:
Revert "Replace anchors with <dfn> for all terms;"

This reverts commit 38441f766f589becd536ed3ab3111014e3ea007f.

dr-shorthair committed:
Revert "test dfn method for x-referencing definitions (from ReSpec)"

This reverts commit 52c7e7d86d81ba9dfc623a72172d069dce957ef2.

dr-shorthair committed:
Fix incorrect header levels

dr-shorthair committed:
Remove items from localBiblio that are now in SpecRef

dr-shorthair committed:
Transparent background on images

TallTed commented on ssn/chapters/Introduction.html:
```diff
@@ -2,61 +2,77 @@
 <p>
   OGC's Sensor Web Enablement standards [[SWE]], [[SensorML]], [[OandM]], [[OMS]], [[STA]] provide
-  conceptual models, as well as some encodings for the annotation of sensors and their observations.
+  conceptual models, and some encodings for the annotation of sensors and their observations.
```
```suggestion
  conceptual models and some encodings for the annotation of sensors and their observations.
```

TallTed replied:
fixed in https://github.com/w3c/sdw-sosa-ssn/pull/341/commits/b96c703ff0597d9e3057f34629e7da59f896c544

TallTed commented on ssn/images/obs-foi-pattern.svg:
```diff
@@ -1,4 +1,4 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg xmlns="http://www.w3.org/2000/svg" style="background: transparent; background-color: transparent; color-scheme: light dark;" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="888px" height="1891px" viewBox="-0.5 -0.5 888 1891" content="&lt;mxfile host=&quot;Electron&quot; agent=&quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/26.0.9 ...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" style="background: transparent; background-color: transparent; color-scheme: light dark;" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="888px" height="1891px" viewBox="-0.5 -0.5 888 1891" content="&lt;mxfile host=&quot;Electron&quot; agent=&quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/27.0.9 ...
```
I can't narrow focus better than this... but there is at least one rectangle in this image that retains the white background.

dr-shorthair replied:
I think I fixed them all in d4d25392cf306e783230521591b488f8985bb83e

TallTed replied:
<img width="475" height="785" alt="Screen Shot 2025-07-11 at 04 14 41 PM" src="https://github.com/user-attachments/assets/3a65e815-9fe5-4c6d-8576-5d5834de7524" />

TallTed commented on ssn/images/historical-airtemp.svg:
```diff
@@ -1,4 +1,4 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg xmlns="http://www.w3.org/2000/svg" style="background: transparent; background-color: transparent; color-scheme: light dark;" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="906px" height="587px" viewBox="-0.5 -0.5 906 587" content="&lt;mxfile host=&quot;Electron&quot; agent=&quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/26.0.9 C ...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" style="background: transparent; background-color: transparent; color-scheme: light dark;" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="906px" height="587px" viewBox="-0.5 -0.5 906 587" content="&lt;mxfile host=&quot;Electron&quot; agent=&quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/27.0.9 C ...
```
Are the black rectangles intentional? They don't seem to be helpful.

dr-shorthair replied:
Can't see them

TallTed replied:
<img width="1106" height="755" alt="Screen Shot 2025-07-11 at 04 11 31 PM" src="https://github.com/user-attachments/assets/f1c274d0-6988-480d-9034-dbc2c862d1dc" />

TallTed commented on ssn/images/ModuleKey.svg:
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Do not edit this file with editors other than draw.io -->
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg xmlns="http://www.w3.org/2000/svg" style="background: transparent; background-color: transparent; color-scheme: light dark;" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="471px" height="381px" viewBox="-0.5 -0.5 471 381" content="&lt;mxfile host=&quot;Electron&quot; agent=&quot;Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/26.0.9 C ...
```
Are these really intended to be black boxes?

dr-shorthair replied:
Unclear

TallTed commented on ssn/images/ModuleKey0.svg:
Are these really intended to be black boxes?

dr-shorthair replied:
Module diagrams still to be polished

TallTed commented on ssn/images/Observation.svg:
There appears to be at least one rectangle in this image that retains the white background.

dr-shorthair replied:
Fixed in d4d25392cf306e783230521591b488f8985bb83e

TallTed replied:
<img width="219" height="96" alt="Screen Shot 2025-07-11 at 04 19 10 PM" src="https://github.com/user-attachments/assets/d9d25c6a-1e95-4284-9fb6-a78bf87f7a64" />

TallTed commented on ssn/images/Sampling.svg:
There appears to be at least one rectangle that still has a white background

dr-shorthair replied:
Fixed in d4d25392cf306e783230521591b488f8985bb83e

TallTed replied:
<img width="116" height="58" alt="Screen Shot 2025-07-11 at 04 20 50 PM" src="https://github.com/user-attachments/assets/ba4eb5d8-8d84-4988-8ac6-8dcea335f768" />

dr-shorthair committed:
Update ssn/chapters/Introduction.html

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

TallTed commented on ssn/chapters/Introduction.html:
```diff
@@ -2,61 +2,77 @@
-  For earlier changes during the development of the SSN Ontology, please consult Annex "D. Change History" in
-  [[vocab-ssn-20171019]].
+  This describes the changes since the 2017 edition [[vocab-ssn-20171019]].
+  Backward compatibility is maintained since descriptions of Actuations, Observations, Samplings and Samples using the
```
```suggestion
  Backward compatibility is maintained since descriptions of Actuations, Observations, Samplings, and Samples using the
```

dr-shorthair committed:
Update ssn/chapters/Introduction.html

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

dr-shorthair committed:
Add w3cid

dr-shorthair commented:
ReSpec errors will clear when #340 and #341 are merged

dr-shorthair committed:
Add note about incomplete OMS extension

dr-shorthair committed:
Merge branch 'gh-pages' into 337-update-front-matter

# Conflicts:
#	ssn/chapters/Actuation.html
#	ssn/chapters/Common.html
#	ssn/chapters/Conformance.html
#	ssn/chapters/OBOE-alignment.html
#	ssn/chapters/OMS.html
#	ssn/chapters/Observation.html
#	ssn/chapters/Sample-Relations.html
#	ssn/chapters/Sampling.html
#	ssn/chapters/System-capabilities.html
#	ssn/images/ModuleKey.svg
#	ssn/images/ModuleKey0.svg
#	ssn/images/historical-airtemp.svg

dr-shorthair committed:
Fix merge issues around Systems-and-their-Deployment section

dr-shorthair committed:
sosa:isMemberOf misnamed in HTML

dr-shorthair committed:
Fix section nesting

dr-shorthair committed:
Better note about missing OMS terms

dr-shorthair committed:
Remove JSON-LD section

dr-shorthair committed:
SickChild --> Sick

dr-shorthair committed:
Add SpecRef link to SSNX alignment text

dr-shorthair committed:
Tidy up alignment text

dr-shorthair committed:
Bad link to diagram

dr-shorthair committed:
Merge branch 'gh-pages' into 337-update-front-matter

ldesousa commented:
Following up on the [comment on PR #338](https://github.com/w3c/sdw-sosa-ssn/pull/338#issuecomment-3092659195), below are two print outs exemplifying the "invisible" diagrams.

<img width="1141" height="255" alt="Selection_092" src="https://github.com/user-attachments/assets/3c3e7c9f-7658-4fe5-ae4a-58f1654e3a1c" />
<img width="1149" height="382" alt="Selection_091" src="https://github.com/user-attachments/assets/0c48ca1d-57f7-46ac-8327-3407b2b78dd8" />

ldesousa commented:
> Following up on the [comment on PR #338](https://github.com/w3c/sdw-sosa-ssn/pull/338#issuecomment-3092659195), below are two print outs exemplifying the "invisible" diagrams.
> 

The print out below is from GIMP. It could be that GitHub is rasterising the SVG in an incorrect way. However, I would still recommend adding a background colour to all "boxes" (classes, individuals, etc).

<img width="1138" height="634" alt="Selection_093" src="https://github.com/user-attachments/assets/4490ff87-e4eb-4557-9173-811016728b97" />

dr-shorthair committed:
Merge branch 'gh-pages' into 337-update-front-matter

# Conflicts:
#	ssn/chapters/OMS.html
#	ssn/chapters/Sampling.html
#	ssn/images/Sampling.svg

dr-shorthair committed:
Rearrange to allow text to be larger

dr-shorthair committed:
Redraw module diagrams in SVG

dr-shorthair committed:
Resize package diagrams

dr-shorthair committed:
Update changelog

dr-shorthair committed:
Replace <code></code> with `backticks` per ReSpec

dr-shorthair committed:
Resize some diagrams (again)

dr-shorthair committed:
Verify all diagram text is black or grey; make line thickness uniform

dr-shorthair committed:
Correct I-ADOPT example per discussion in https://github.com/w3c/sdw-sosa-ssn/issues/347

dr-shorthair commented:
I've made another pass through the figures. 
All text is now black, except the places where it is grey because it labels classes and properties from another module. 
Line thickness is now uniform 2px.

dr-shorthair committed:
HTML formatting

dr-shorthair committed:
HTML formatting

dr-shorthair committed:
Merge remote-tracking branch 'origin/337-update-front-matter' into 337-update-front-matter

dr-shorthair commented:
Sorry about all the changes to so many files. Many of them are just minor adjustments due to 
(i) rationalizing the figure id's 
(ii) replacing \<code\> tags with back-ticks \`\`, per ReSpec support
(iii) a big pass through the diagrams to make the text and linework consistent. 

Probably the chapters that you should actually review are 

- config.js - I moved all the regular telecon attendees to 'Editor' status, and most of the bibliography out to SpecRef  https://github.com/w3c/sdw-sosa-ssn/pull/341/files#diff-86ae4ad426820d63785855ad091cfcce9475152af8fe90b8e8802a57832d1d44 
- index.html - I updated the changelog  https://github.com/w3c/sdw-sosa-ssn/pull/341/files#diff-c71496241ac09b84e36c3c74d5e68a5e10f975b4bca1cf165cc5d71235d9d9b2 
- Indroduction.html - various small tweaks https://github.com/w3c/sdw-sosa-ssn/pull/341/files#diff-da3fb2750231e40bedc046fdf2a8ffba528ead07811076c4c7a0aab4a2ef01cf 
- OMS.html https://github.com/w3c/sdw-sosa-ssn/pull/341/files#diff-0de0f1a8ad580595bf1e2d2b821e8d5e9aa2c2409392633911f07927fc84435c 

and look at the preview to see it all put together https://raw.githack.com/w3c/sdw-sosa-ssn/337-update-front-matter/ssn/index.html

ldesousa commented on ssn/chapters/Common.html:
```diff
@@ -2364,16 +2375,17 @@ <h6><dfn>sosa:System</dfn></h6>
                 <em>deployed asset</em> &mdash;
-                relation from a Deployment to an asset (System or Platform)
+                Relation between a <a href="#SOSADeployment">Deployment</a>
+                and a deployed asset (<a href="#SOSASystem">System</a> or <a href="#SOSAPlatform">Platform</a>).
```
Without looking at the full text, shouldn't `Sampler` be here too?

dr-shorthair replied:
`System` is the superclass of `Actuator`, `Sensor`, and `Sampler`. 
So either all of the subclasses should be mentioned, or none. 
Note that the link to the definition of System should take the reader to understand that there are subclasses. 

This is in fact a tension throughout the *Common* module. 
In many places we mention the sub-classes of Procedure, System, Execution, ExecutionCollection parenthetically, but in some cases not. 
I'm willing to be guided by the consensus of the editorial team on which direction to fall on this. 

A lot of the minor revisions have been my attempt to make the whole document more internally consistent, but there is plenty more work to be done.

ldesousa commented on ssn/chapters/Introduction.html:
```diff
@@ -2,61 +2,77 @@
-  interpreting sensor data requires more than just the sensing result.
-  For proper interpretation of these values, knowledge of the feature of interest, such as a river, the observed property, such as flow velocity, the sampling strategy, such as the specific locations and times at which the velocity was measured, and a variety of other information are required.
+  Sensors are a major source of data available on the Web today.
+  However, searching, reusing, integrating, and interpreting sensor data requires more than just the sensing result.
```
```suggestion
  However, searching, reusing, integrating, and interpreting sensor data requires more than just the sensed result.
```

dr-shorthair replied:
Changed to 'result of an act of sensing'

ldesousa commented on ssn/chapters/ModelComplexSystem.html:
```diff
@@ -3,16 +3,16 @@ <h3>Complex devices</h3>
     <embed alt="Complex sensor package" src="images/Inkbird-system.svg" style="width:100%" />
     <figcaption>Inkbird model IBS-TH2-PLUS temperature and humidity sensor package represented with both package
-        and individual sensors represented as instances of sub-classs of <a href="#SOSASystem"><code>sosa:System</code></a> and <a href="#SOSASensor"><code>sosa:Sensor</code></a>.
+        and individual sensors represented as instances of sub-classs of <a>sosa:System</a> and <a>sosa:Sensor</a>.
```
```suggestion
        and individual sensors, the former represented as an instance of sub-classs of <a>sosa:System</a> and the later of <a>sosa:Sensor</a>.
```

dr-shorthair replied:
Changed to 

> with the individual sensors represented as instances of sub-classes of <a>sosa:Sensor</a>, 
> and the overall package represented as an instance of a sub-classs of <a>sosa:System</a>

ldesousa commented on ssn/images/Actuation-overview.svg:
I am approving all images as they are, even though GitHub is not rendering them correctly.

dr-shorthair replied:
Yes - the different systems seem to render differently, unfortunately. 
I've been verifying using the final rendered HTML ...

dr-shorthair committed:
Responses to https://github.com/w3c/sdw-sosa-ssn/pull/341#pullrequestreview-3043464290

dr-shorthair committed:
Align definitions in HTML with those in RDF/TTL

dr-shorthair committed:
Tweak Procedure definition and notes
